deepspeed/ops/transformer/inference/transformer_inference.py

```diff
@@ -146,6 +146,31 @@ def _transpose_for_context(x):
                                       (hidden_size_per_partition,)
             return x.view(*new_x_layer_shape)
 
+        def backup_attention(mixed_query, key_layer, value_layer, input_mask):
```

Comment:
Is this part of the fix?

Reply:
no it is not, not sure how it got there! I will remove it